scripts/metrics.py

```diff
@@ -244,11 +252,11 @@
                               lisi_graph_= lisi_graph_,
                               trajectory_=trajectory_
                              )
-    results.rename(columns={results.columns[0]:out_prefix}, inplace=True)
+    results.rename(columns={results.columns[0]:setup}, inplace=True)
     if verbose:
         print(results)
     # save metrics' results
-    results.to_csv(os.path.join(args.output, f'{out_prefix}.csv'))
+    results.to_csv(args.output)
```

Comment:
why do you no longer need the filename here?

Reply:
script now expects filename as input instead of directory. Should be the same as for any other rule